### PR TITLE
added standby CEPH-11345 & CEPH-11259

### PR DESCRIPTION
**Pull request by yogesh-mane**
yogesh@Yogeshs-MacBook-Pro

# Description

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

mds service add/remove logs - http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-JRV3OH
latest mds logs - http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-XV26RS

mon service add/remove logs - http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-5KOJ5W
latest mon logs - http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-PBCI6H

### CODE REVIEW

**Review by hkadam134 — APPROVED** [review]
LGTM

**Comment by hkadam134** [comment]
@yogesh-mane - I don't see MON service rm add logs ? can you reverify the run and add proper logs?

**Comment by yogesh-mane** [comment]
Hi @hkadam134 , If you see "removing mon" in logs below that we will be adding 2 mons instead of 3 mons. That's how we are removing mons. 'ceph orch  --verbose apply mon cephfs --placement='ceph-ymane-53-bug-j3uzaw-node1-installer ceph-ymane-53-bug-j3uzaw-node2'

**Comment by yogesh-mane** [comment]
I think I attached old logs, please check the logs again. I have update correct log link

File: tests/cephfs/mon_rm_add.py [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+        client.exec_command(sudo=True, cmd=cmd)
+        wait_for_cmd(client, "mon", count)
+        mons_hosts = " ".join([str(elem) for elem in mon_hosts])
+        cmd = f"ceph orch  --verbose apply mon cephfs --placement='{mons_hosts}'"
+        client.exec_command(sudo=True, cmd=cmd)
+        wait_for_cmd(client, "mon", initial_mon_count)
+        stop_flag = True
+
+    except Exception as e:
+        log.info(e)
```
Comment (AmarnatReddy) [inline tests/cephfs/mon_rm_add.py:78]: Can you make this to log.error() in all the places in exception block

**Review by hkadam134 — APPROVED** [review]
LGTM, 

@yogesh-mane - Please submit the latest changes as per Amar's suggestion.

**Review by AmarnatReddy — APPROVED** [review]
LGTM

**Pull request merged**